### PR TITLE
Feature/add  pragma support

### PR DESCRIPTION
**Pull request by jordr**
Resolves https://github.com/eliben/pycparser/issues/485 for [_Pragma](https://gcc.gnu.org/onlinedocs/cpp/Pragmas.html) support.
After the preprocessing stage, some `_Pragma`s may remain. This at least lets us parse such code. 

I'm not certain about the `_token_coord(p, 2))` here though. Might be 3?
https://github.com/statinf-software/pycparser/blob/32b35034bf06d76af10054da8cc227b9e6dd75dd/pycparser/c_parser.py#L579-L580

(PS: I get `ModuleNotFoundError: No module named '_ast_gen'` when trying to run the tests, but I think the test additions still ran and passed.)

### CODE REVIEW

File: pycparser/c_parser.py [hunk 1]
```diff
@@ -574,8 +574,11 @@ def p_pp_directive(self, p):
     def p_pppragma_directive(self, p):
         """ pppragma_directive      : PPPRAGMA
                                     | PPPRAGMA PPPRAGMASTR
+                                    | _PRAGMA LPAREN unified_string_literal RPAREN
```
Comment (eliben) [inline pycparser/c_parser.py:577]: Can you add a comment explaining why the contents of `_PRAGMA` differ from `PPPRAGMA`?

**Review by eliben — CHANGES_REQUESTED** [review]
Thank you; please rebase this PR on top of the current master - it had a commit that should fix action failures on old Python versions

File: pycparser/c_parser.py (hunk 1 above)
Comment (jordr) [reply pycparser/c_parser.py:577]: Well, it has a different syntax? 
image

`#pragma` does not require parenthesis and strings as far as I know. Not sure what to explain?

Comment (eliben) [reply pycparser/c_parser.py:577]: The comment is good, thanks! Please move it out of the docstring though - this doesn't work with PLY's yacc (see test error in the GitHub action)

**Review by eliben — CHANGES_REQUESTED** [review]

**Comment by jordr** [comment]
Thanks for the help! Looks good to me now.

**Review by eliben — APPROVED** [review]

**Pull request merged**